index.html

```diff
@@ -840,6 +841,10 @@ <h3>
         <p>
           Unless otherwise specified, the <a>JavaScript realm</a> for script
           objects constructed by algorithm steps is the <a>current realm</a>.
+          Unless otherwise specified, all global tasks use the <a data-cite=
+          "!HTML#networking-task-source">networking task source</a> associated
+          with the <a data-cite="!HTML#global-object">global object</a> of the
+          <a>current realm</a>.
```

Comment:
Use of the "networking task source" contradicts the previous paragraph, which defines a dedicated "presentation task source".

I think it is good practice to create a dedicated task source unless there's a risk that tasks conflict with other tasks defined in other specs. I don't see any reason why Presentation API tasks would conflict with other networking tasks and require relative ordering with them.

Reply:
I forgot that the spec defines its own presentation task source.  Will remove the reference to the networking task source.